lib/parse.js

```diff
@@ -1320,11 +1320,17 @@ function parse($TEXT, options) {
             var type = start.type;
             var name = as_property_name();
             if (type == "name" && !is("punc", ":")) {
+                var createAccessor = function() {
```

Comment:
I'm confused - did you try my suggestion or yours? Yours has an extra `function() {...}` wrapped around it.